packages/i18next/src/lib/functions.ts

```diff
@@ -0,0 +1,181 @@
+
+/**
+ * Send a localized message using an objects option.
+ * @since 2.0.0
+ * @param target The target to send the message to.
+ * @param options The options to be sent, requiring at least `keys` to be passed.
+ * @example
+ * ```typescript
+ * await sendLocalized(message, { keys: 'commands/ping:loading' });
+ * // ➡ "Pinging..."
+ * ```
+ * @example
+ * ```typescript
+ * const latency = 42;
+ *
+ * await sendLocalized(message, {
+ * 	keys: 'commands/ping:loading',
+ * 	formatOptions: { latency }
+ * });
+ * // ➡ "Pinging... current latency is 42ms."
+ * ```
+ */
+export async function sendLocalized<TKeys extends TFunctionKeys = string, TInterpolationMap extends NonNullObject = StringMap>(
+	target: Target,
+	options: LocalizedMessageOptions<TKeys, TInterpolationMap>
+): Promise<Message>;
+export async function sendLocalized<TKeys extends TFunctionKeys = string, TInterpolationMap extends NonNullObject = StringMap>(
+	target: Target,
+	options: TKeys | TKeys[] | LocalizedMessageOptions<TKeys, TInterpolationMap>
+): Promise<Message> {
+	const channel = _getChannel(target);
+	return channel.send(await _resolveOverloads(target, options));
+}
+
+/**
+ * Edits a message using the language key only.
+ * @since 2.0.0
+ * @param target The target to send the message to.
+ * @param options The options to be sent, requiring at least `keys` to be passed.
+ * @example
+ * ```typescript
+ * await sendLocalized(message, 'commands/ping:success');
+ * // ➡ "Pong!"
+ * ```
+ */
+export async function editLocalized<TKeys extends TFunctionKeys = string>(target: Message, keys: TKeys | TKeys[]): Promise<Message>;
+
+/**
+ * Edits a message using an objects option.
+ * @since 2.0.0
+ * @param target The target to send the message to.
+ * @param options The options to be sent, requiring at least `keys` to be passed.
+ * @example
+ * ```typescript
+ * await editLocalized(message, { keys: 'commands/ping:success' });
+ * // ➡ "Pong!"
+ * ```
+ * @example
+ * ```typescript
+ * const latency = 42;
+ * const took = 96;
+ *
+ * await editLocalized(message, {
+ * 	keys: 'commands/ping:success',
+ * 	formatOptions: { latency, took }
+ * });
+ * // ➡ "Pong! Took me 96ms to reply, and my heart took 42ms to beat!"
+ * ```
+ */
+export async function editLocalized<TKeys extends TFunctionKeys = string, TInterpolationMap extends NonNullObject = StringMap>(
+	target: Message,
+	options: LocalizedMessageOptions<TKeys, TInterpolationMap>
+): Promise<Message>;
+export async function editLocalized<TKeys extends TFunctionKeys = string, TInterpolationMap extends NonNullObject = StringMap>(
+	target: Message,
+	options: TKeys | TKeys[] | LocalizedMessageOptions<TKeys, TInterpolationMap>
+): Promise<Message> {
+	return target.edit(await _resolveOverloads(target, options));
+}
+
+async function _fetchLanguage(context: InternationalizationContext): Promise<string> {
```

Comment:
You know I don't like underscore prefixes... :( 

Can you rename this to something else? Same for `_getChannel` and `_resolveOverloads`

Or move them to `src/lib/internals.ts` and don't export them on the root `index.ts`.